app/(main)/league/[leagueId]/entry/all/page.test.tsx

```diff
@@ -312,7 +312,6 @@ describe('League entries page (Entry Component)', () => {
         selectedTeams: ['Packers'],
       },
     ]);
-
```

Comment:
Revert this change.

Reply:
not seeing that space locally. might be fixed

Reply:
Please address this feedback in your code.